tests/basic_setters.rs

```diff
@@ -24,15 +24,15 @@ impl Lorem {
 fn contructor_sanity_check() {
     let x = Lorem::new("lorem");
 
-    assert_eq!(x, Lorem { ipsum: "lorem".into(), dolor: "".into(), sit: 0, amet: false, });
+    assert_eq!(x, Lorem { ipsum: "lorem".into(), dolor: None, sit: 0, amet: false, });
 }
 
 #[test]
 fn setters() {
-    let x = Lorem::new("lorem")
-        .dolor("dolor")
-        .sit(42)
-        .amet(true);
+    let mut x = Lorem::new("lorem");
+    x.dolor(Some("dolor".into()));
+    x.sit(42);
+    x.amet(true);
```

Comment:
Funny, if there was a reason I have forgotten it by now. I'll change it back. :-)